README.md

```diff
@@ -227,6 +227,6 @@ Where can I download node JS for installation?
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](https://github.com/MarwanDev/leaderboard/blob/project-setup/LICENSE) licensed.
+This project is [MIT](https://github.com/citec-47/Leader-board/blob/leader-board/LICENSE) licensed.
```

Comment:
- [ ] Your README is somewhat noisy and contains a lot of irrelevant information. I would recommend you remove the linter installation sections:

  ![image](https://user-images.githubusercontent.com/36542986/231413389-648d97be-29e9-4067-9464-2cf021980f9e.png)